### PR TITLE
Create prerelease.yml

### PR DESCRIPTION
**Pull request by arjuncooliitr**
Created prerelease.yml

## Description

This workflow is added to simplify the work around prerelease publishing to npm and update package.json with an npm pre-release version

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by arjuncooliitr** [comment]
image

Saying there's problem with the syntax. It may be on the line where I am adding comma-separated dependency list. Not sure if that is wrong.

File: .github/workflows/prerelease.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        required: false
+        default: 'pre'
+      dist-tag:
+        description: 'The dist-tag use'
+        required: false
+        default: 'next'
+      dependencies-to-update:
+        description: 'csv of dependencies to update with the dist-tag'
+        required: false
+        default: '@adobe/aio-lib-runtime','@adobe/aio-lib-web'
```
Comment (shazron) [inline .github/workflows/prerelease.yml:16]: the value should be a one string csv: '@adobe/aio-lib-runtime,@adobe/aio-lib-web'

Comment (arjuncooliitr) [reply .github/workflows/prerelease.yml]: Okay, fixed that

File: .github/workflows/prerelease.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        required: false
+        default: 'pre'
+      dist-tag:
+        description: 'The dist-tag use'
+        required: false
+        default: 'next'
+      dependencies-to-update:
+        description: 'csv of dependencies to update with the dist-tag'
+        required: false
+        default: '@adobe/aio-lib-runtime,@adobe/aio-lib-web'
```
Comment (shazron) [inline .github/workflows/prerelease.yml:16]: missing the other @adobe deps: https://github.com/adobe/aio-cli-plugin-app/blob/cb1b5c05d4c9afaa7ee57a1b96c5af26c31db4b1/package.json#L8-L17

**Review by shazron — CHANGES_REQUESTED** [review]

File: .github/workflows/prerelease.yml (hunk 2 above)
Comment (arjuncooliitr) [reply .github/workflows/prerelease.yml]: Added all @adobe deps for this plugin

**Review by shazron — APPROVED** [review]

**Pull request merged**